doc/lua_api.txt

```diff
@@ -5644,6 +5644,16 @@ Environment access
       active config.
 * `minetest.get_noiseparams(name)`
     * Returns a table of the noiseparams for name.
+* `minetest.get_containing_chunk(blockpos, [chunksize])`
+    * Returns the minimum block coordinate of the chunk containing `blockpos`.
+    * `chunksize` is the optional chunk side length in blocks. It defaults to
+      the `"chunksize"` mapgen setting.
+* `minetest.get_mapgen_area_blocks([mapgen_limit], [chunksize])`
```

Comment:
i don't have a suggestion for a better name but I think it's really non-obvious

Reply:
Maybe just `get_mapgen_area`?

Reply:
I think I'll go with `get_mapgen_bounds`.